pkg/e2e/autoscaler/autoscaler.go

```diff
@@ -372,5 +377,15 @@ var _ = g.Describe("[Feature:Machines][Serial] Autoscaler should", func() {
 				remaining(testDuration), len(nodes), len(currentNodes))
 			return len(currentNodes)
 		}, e2e.WaitMedium, pollingInterval).Should(o.Equal(len(nodes)))
+
+		g.By("Waiting for scaled up machines to be deleted")
```

Comment:
what was the motivation for this addtion?

Reply:
Since it takes longer to delete a vm in Azure.